qiskit_metal/analyses/sweep_options/sweeper.py

```diff
@@ -0,0 +1,210 @@
+from qiskit_metal import Dict
```

Comment:
Different folder name maybe make more sense? optimize_and_sweep ? Not a big deal of course.